docker-compose.yml

```diff
@@ -4,28 +4,32 @@ volumes:
     postgres_data:
 
 services:
-    elasticsearch:
+    elasticsearch2:
         build:
             context: ./
             dockerfile: ./docker/elasticsearch/Dockerfile
         ports:
-            - "9200:9200"
-            - "9300:9300"
-    elasticsearch7:
-        image: elasticsearch:7.4.2
-        ports: 
             - "9000:9200"
             - "9100:9300"
+    elasticsearch:
+        build:
+            context: ./
+            dockerfile: ./docker/elasticsearch/7.4/Dockerfile
+        ports: 
+            - "9200:9200"
+            - "9300:9300"
```

Comment:
You've swapped the way these ports are configured between the ES2 and ES7 services. Is that intentional?

Reply:
Yes, due to the alpine kibana image assuming configuration and not being able to overwrite it (looks like a bug as their documentation indicates you should be able to overwrite the ES endpoint but haven't seen it work yet)